gnd/src/main/java/com/google/android/gnd/model/observation/Record.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.model.observation;
+
+import androidx.annotation.Nullable;
+import com.google.android.gnd.model.Project;
+import com.google.android.gnd.model.Timestamps;
+import com.google.android.gnd.model.feature.Feature;
+import com.google.android.gnd.model.form.Form;
+import com.google.android.gnd.system.AuthenticationManager.User;
+import com.google.auto.value.AutoValue;
+
+/** Represents a single instance of data collected about a specific {@link Feature}. */
+@AutoValue
+public abstract class Record {
```

Comment:
It should be but saving renaming for a different PR if it's ok, since it'll be a much bigger change. Thought I'd start off with the package to reduce the churn.